corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js

```diff
@@ -435,7 +435,7 @@ hqDefine('app_manager/js/forms/advanced/case_config_ui', function () {
                             path: '',
                             key: 'name',
                             required: true,
-                            save_only_if_edited: action.name_update.update_mode === 'edit',
+                            save_only_if_edited: false,
```

Comment:
This is because at this point, `action` is just [this display/value object](https://github.com/dimagi/commcare-hq/blob/fc70c774c2c17b935bd0431a95eb54f92efcdeca/corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_config_ui.js#L365-L368), right?

Reply:
yes, that's right